BookReader/BookReader.js

```diff
@@ -1000,13 +1000,14 @@ BookReader.prototype.drawLeafsThumbnail = function(seekIndex) {
                 link = document.createElement("a");
                 $(link).data('leaf', leaf);
                 link.addEventListener('mouseup', function(event) {
-                  self.updateFirstIndex($(this).data('leaf'));
+                  self.updateFirstIndex($(this).data('leaf'), true);
```

Comment:
could be useful to have a variable for `suppressFragmentChange = true` so it's more immediately clear why the `true` is necessary in this case.

approach lgtm!